ui/app/routes/variables/new.js

```diff
@@ -1,10 +1,11 @@
 import Route from '@ember/routing/route';
 
 export default class VariablesNewRoute extends Route {
-  model() {
-    return this.store.createRecord('variable');
+  model(params) {
+    return this.store.createRecord('variable', { path: params.path });
```

Comment:
Nice little shortcut for when you are creating a new variable from within an existing directory; it pre-fills the path field and focuses your cursor after the last slash.